pegasus/sites.v3/hourofcode.com/public/css/front-page.css

```diff
@@ -96,6 +96,52 @@ a.category-link {
   margin-right: 10px;
 }
 
+.banner-box {
+  width: 100%;
+  height: fit-content;
```

Comment:
note that fit-content is not supported by IE

Reply:
Removed.